### ISSUE TITLE
Can not reproduce linear probe result

### ISSUE THREAD

**Issue by JunlinHan** [facebookresearch/asym-siam#4]
Hello!
Thanks for your wonderful work and code release!

I tried to run the the linear probe with the released scale_mix checkpoint on 8 GPUs and 4 GPUs. In either GPU setting, without modifying any training settings, I get a final top-1 accuracy around 67.1% only (with the best result ~ 67.2%), lower than the reported 67.6%.

I'm wondering, how many GPUs you used for linear probe? Do you have any ideas regarding th potential reasons for inconsistent?

Many thanks for your reply and help!

**Comment by haooooooqi** [comment]
Hi @JunlinHan,

Thanks for trying out our asym-siam work. 
Please let me know if this command works from your side. (I have re-downloaded the ScaleMix checkpoint to re-verify the performance) 
`python ./asymm/main_lincls.py -a resnet50 --dist-url 'tcp://localhost:10001' --dist-url 'tcp://localhost:10001' --multiprocessing-distributed --world-size 1 --rank       0 /datasets01/imagenet_full_size/061417 --lr 90.0 --cos`

**Comment by JunlinHan** [comment]
> Hello Haoqi @haooooooqi 
Thanks for your reply!
I guess I have fitured out the reason:
![图片](https://user-images.githubusercontent.com/48272064/165915322-67295fb7-e440-49e5-9fda-bde058a267df.png)

The result reported in the paper is actually 67.3, which is close to my re-produced result 67.1-67.2

Would be great if the result in this repo can be updated!

**Comment by haooooooqi** [comment]
<img width="1160" alt="Screen Shot 2022-04-29 at 10 02 37 AM" src="https://user-images.githubusercontent.com/3500423/165990447-4cdb126a-ea34-4465-8923-4856dc00fc56.png">

**Comment by JunlinHan** [comment]
> <img alt="Screen Shot 2022-04-29 at 10 02 37 AM" width="1160" src="https://user-images.githubusercontent.com/3500423/165990447-4cdb126a-ea34-4465-8923-4856dc00fc56.png">

Oh I see, I will test it again. Will update my results then.

**Comment by JunlinHan** [comment]
> Hi @JunlinHan,
> 
> Thanks for trying out our asym-siam work. Please let me know if this command works from your side. (I have re-downloaded the ScaleMix checkpoint to re-verify the performance) `python main_lincls.py -a resnet50 --lr 90.0` (You should expect to get 67.6 with this line)

Hi Haoqi,
With SGD optimizer and lr = 90 (python main_lincls.py -a resnet50 --lr 90.0), I can get a final result ~67.42 (with best result ~67.46), pretty close to 67.6%.

When changing batch_size to 256 (SGD+lr 90), the result will end up with ~68%.

The 67.3 result (reported in paper) can be reproduced by Lars optimizer and batch_size = 4096. The 67.6 result reported in this repo can be reproduced by using SGD + large lr. 

Many thanks for your help!
